include/arch/x86/page.h

```diff
@@ -196,6 +212,19 @@ static inline void *kmap(mfn_t mfn, unsigned int order,
                 l3_flags, l2_flags, l1_flags);
 }
 
+static inline void *mmap(mfn_t mfn, unsigned int order,
```

Comment:
Same here
name `mmap` is misleading.

Reply:
Ok. Suggestions welcome.

Reply:
With VIRT_KERNEL_MAP I hope it's not ok.